x/gov/abci.go

```diff
@@ -30,20 +30,19 @@ func EndBlocker(ctx sdk.Context, keeper *keeper.Keeper) error {
 			// if the proposal has an encoding error, this means it cannot be processed by x/gov
 			// this could be due to some types missing their registration
 			// instead of returning an error (i.e, halting the chain), we fail the proposal
-			if errors.Is(err, collections.ErrEncoding) {
```

Comment:
Personally I find the current if statement in this file easier to read than those changes.